### ISSUE TITLE
Es6 demo or example instead of typescript

### ISSUE THREAD

**Issue by vamsivarikuti** [Urigo/angular2-meteor#32]
Readme.md says, Es6 modules are supported. Is there an example or demo that we can follow to use es6 with angular2

Will the ecmascript package in meteor handle client side angular2?

**Comment by barbatus** [comment]
You'll need decorators plugin added to Babel.
Current package configures Babel to use decorators, so you won't need Meteor's  ecmascript package.

Babel actually has some limitations regarding Angular2 (for example, it doesn't emit decorators metadata yet, which Angular2 heavily relies on). Though, it's possible to overcome this, code doesn't look that nice as TypeScript one.  So we are now kinda deciding whether to postpone Babel for time being. If we decide to continue support, I'll add a demo.

**Comment by Urigo** [comment]
Follow support for decorators in Meteor here - https://github.com/meteor/meteor/issues/6028

We still recommend Typescript as the compiler for Angular2-Meteor apps

**Comment by shotleybuilder** [comment]
Hi.  Somewhat confused about what the current standard platform provides without custom tinkering.  I noticed the tsconfig.json is preset by angular2-meteor to ES3 and this cannot be changed.  I think this is the reason that my ng2 code that uses a few, not many, ES6 things like accessors and promises failed.
I had to reinstall the meteor ecmascript package to get things to run too.
My understanding, correct me if this is wrong, is that Meteor1.3 and ng2 both support ES6.
Also, Atom doesn't complain about my use of ES6, it's only is the console that the errors are logged.  I'm not sure if I should expect Atom to highlight these things.
